app/lib/frontend/templates/views/pkg/versions/version_row.mustache

```diff
@@ -4,6 +4,11 @@
 
 <tr data-version="{{version}}">
   <td class="version"><a href="{{& version_url}}">{{version}}</a></td>
+  <td class="sdk">
+    {{#has_sdk}}
+    <code>{{ sdk }}</code>
+    {{/has_sdk}}
+    </td>
```

Comment:
Just a question: If it doesn't have an SDK does the SDK cell(td) even need to be included at all? Can the conditional be lifted up?

Either way closing `td` tag here can be unindented.

Reply:
The most visited pages are likely to be modern packages which do have the SDK constraint and thus they will have the column there. While we could make the entire column hidden if none of the versions have constraint, the extra calculation and conditions will yield very little overall.